### ISSUE TITLE
[Feature Request] List all your uploaded avatar's IDs

### ISSUE THREAD

**Issue by BLANKE1** [pypy-vrc/VRCX#205]
A feature like this would be beneficial for things like mass settings avatar IDs on pedestals with [VRWorldToolKit](https://github.com/oneVR/VRWorldToolkit) an ability to filter between public and private will be appreciated aswell

**Comment by Murkur** [comment]
I agree would like this aswell as for it to show there publication status i know its possible just needs to filter out useless json extras

**Comment by Natsumi-sama** [comment]
I have already made avatar ID's more visible with this commit https://github.com/pypy-vrc/VRCX/commit/34efa9f24b3ab5450e075ed8a474f60e31bc6788 ![Image](https://user-images.githubusercontent.com/11171153/118881487-40729600-b947-11eb-80e8-9057e30eb485.png) but if you still want an export list I will add it.

**Comment by Murkur** [comment]
would still like the export list and on another note for commit 34efa9f you should Hide the avatar id behind a hidden element unless selected if not already done [hide text unless clicked](https://stackoverflow.com/questions/47340492/how-to-make-html-text-hidden-until-clicked-on)

**Comment by Natsumi-sama** [comment]
Why would it need to be hidden, I don't see any benefit in that anymore.

**Comment by Murkur** [comment]
Entire point of the api update was to stop people you didnt want to get avatars you didn't want to share with them that includes publics maybe your streaming on discord and happen to select one of your avatars or another's on accident?

**Comment by Natsumi-sama** [comment]
https://github.com/pypy-vrc/VRCX/commit/857f2f48141993ee6e8835769faeb336111b554e
